src/newSeed/baseStage.scss

```diff
@@ -499,6 +499,9 @@
     > .heading.heading2 {
       font-size: 26px;
     }
+    label {
+      line-height: 1.2em;
```

Comment:
Now there is insufficient margin below the labels

![image](https://user-images.githubusercontent.com/1821666/123637261-0a90bf00-d7db-11eb-921a-bda75863333b.png)

Reply:
How is this space ?
![Screenshot 2021-06-28 at 13 40 13](https://user-images.githubusercontent.com/19796652/123637838-5f9df680-d816-11eb-852e-de0b037693fd.png)

Reply:
Added a margin-bottom to the labels